src/cleanlab_tlm/utils/rag.py

```diff
@@ -131,6 +132,18 @@ def __init__(
 
             self._evals = evals
 
+        self._validate_disable_trustworthiness_option(options)
+
+    def _validate_disable_trustworthiness_option(self, options: Optional[TLMOptions]) -> None:
```

Comment:
I would move this to the `internal.validation` module instead of a class method here